testdata/visualize.py

```diff
@@ -0,0 +1,66 @@
+# Simple peers graph visualization using rustworkx and matplotlib (default rendering backend)
+# Preqrequisites:
+# pip install rustworkx
+# pip install matplotlib
+
+import json
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+import rustworkx as rx
+from rustworkx.visualization import mpl_draw
+import sys
+
+conn_limit = 1000
+
+if len(sys.argv) < 2:
+    print("Usage: python main.py peers.json")
+    exit(1)
+
+f = open(sys.argv[1])
+data = json.load(f)
+
+graph = rx.PyGraph()
+
+ipdict = {}
+
+for node in data:
+    idx = graph.add_node(node['ip'])
+    ipdict[node['ip']] = idx
+
+conns = 0
+for node in data:
+    node_id = ipdict[node['ip']]
+    for peer_ip in node['list']:
+        graph.add_edge(node_id, ipdict[peer_ip], None)
```

Comment:
should the `conns` be set to zero before we start checking peers here for each node? 🕵️‍♀️

Reply:
Fixed. For small number of connections selected it has not generated bad results that's why missed it.

Reply:
> Fixed

Actually, can you check if you pushed that change, maybe `git add` didn't include that file

Reply:
Should be pushed right now.